SwiftMessageBar/SwiftMessageBar.swift

```diff
@@ -19,7 +19,8 @@ public struct MessageBarConfig {
   
   public init(errorColor: UIColor = UIColor.red, successColor: UIColor = .green, infoColor: UIColor = .blue,
               titleColor: UIColor = .white, messageColor: UIColor = .white, statusBarHidden: Bool = false,
-              successIcon: UIImage? = nil, infoIcon: UIImage? = nil, errorIcon: UIImage? = nil) {
+              successIcon: UIImage? = nil, infoIcon: UIImage? = nil, errorIcon: UIImage? = nil,
+              languageDirection: NSLocale.LanguageDirection = .leftToRight) {
```

Comment:
Yes, you are absolutely right, default should be '.unknown'. But i think the 'languageDirection' is fine. Its just an option for the caller, if they don't pass it we are not forcing it.